test/integration/targets/vmware_guest_network/tasks/main.yml

```diff
@@ -73,6 +73,61 @@
         - "del_netadapter.changed == true"
         - "{{ del_netadapter.network_data | length | int }} == {{ netadapter_num | int + 1 }}"
 
+  - name: get instance uuid of virtual machines
+    vmware_guest_info:
+      validate_certs: False
+      hostname: "{{ vcenter_hostname }}"
+      username: "{{ vcenter_username }}"
+      password: "{{ vcenter_password }}"
+      name: "{{ virtual_machines[0].name }}"
+      datacenter: '{{ dc1 }}'
+    register: guest_info
+
+  - set_fact: vm1_instance_uuid="{{ guest_info['instance']['instance_uuid'] }}"
+
+  - name: add new network adapters to virtual machine with instance uuid
+    vmware_guest_network:
+      validate_certs: False
+      hostname: "{{ vcenter_hostname }}"
+      username: "{{ vcenter_username }}"
+      password: "{{ vcenter_password }}"
+      uuid: "{{ vm1_instance_uuid }}"
+      use_instance_uuid: True
+      networks:
+        - name: "VM Network"
+          state: new
+          device_type: e1000e
+          manual_mac: "00:50:56:58:59:60"
+    register: add_netadapter_instanceuuid
+
+  - debug: var=add_netadapter_instanceuuid
+
+  - name: assert the new netowrk adapters were added to VM
+    assert:
+      that:
+        - "add_netadapter_instanceuuid.changed == true"
```

Comment:
You can use the `is` operator here: `add_netadapter_instanceuuid.changed is true`

Reply:
Done